LayoutTests/fast/events/pointer/ios/drag-populates-pointer-events-with-movementxy-fields-expected.txt

```diff
@@ -0,0 +1,31 @@
+Test that the movementX/Y fields of the pointer events fired from a touch drag are correct.
+
+On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".
+
+
+
+    pointerup fired.
```

Comment:
This should be `pointerdown`

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/eed593fcf230c5752fc982cef0b7cbdda782897a